tfe/resource_tfe_organization_token.go

```diff
@@ -35,6 +35,13 @@ func resourceTFEOrganizationToken() *schema.Resource {
 				Computed:  true,
 				Sensitive: true,
 			},
+
+			"keepers": {
```

Comment:
I don't think this belongs in this PR

Reply:
ugh, sorry. I branched off the previous PR I was reviewing. I am going to rebase.